packages/in_app_purchase/example/android/app/build.gradle

```diff
@@ -6,6 +6,41 @@ if (localPropertiesFile.exists()) {
     }
 }
 
+// Load the build signing secrets from a local `keystore.properties` file.
+// TODO: Create release keys and a `keystore.properties` file. See `example/README.md` for more info.
+def keystorePropertiesFile = rootProject.file("keystore.properties")
+def keystoreProperties = new Properties()
+def configured = true
```

Comment:
This looks like quite a lot setup to require from each up to manually do in their gradle file.
Would be nice if we could move as much as we can to be part of the plugin's gradle config.

Totally doesn't need to happen in this PR, and totally ok to start with this so we can get-going on development.

Reply:
This part really needs to be done by each developer individually for the APIs to work. I did add an example `keystore` file to the repo though to try and make it a little easier to follow.